test/parallel/test-eventemitter-asyncresource.js

```diff
@@ -130,3 +131,46 @@ function makeHook(trackedTypes) {
     ],
   ]));
 })().then(common.mustCall());
+
+// Member methods ERR_INVALID_THIS
+{
+  throws(
+    () => {
+      EventEmitterAsyncResource.prototype.emit();
+    },
+    {
+      code: 'ERR_INVALID_THIS',
+    }
```

Comment:
Can we limit the whitespace here and below please?
```suggestion
    () => EventEmitterAsyncResource.prototype.emit(),
    { code: 'ERR_INVALID_THIS' }
```